RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -224,20 +280,66 @@
 	),
     vertices = 'firstStepGoodPrimaryVertices'
 )
+jetCoreRegionalStepBarrel = jetCoreRegionalStep.clone(
+    src = 'jetCoreRegionalStepBarrelTracks',
+    mva = dict(
+	minPixelHits  = [1,1,1], # they could be easily increased to at least 2 or 3 !
+        maxChi2       = [9999.,9999.,9999.],
+        maxChi2n      = [1.6,1.0,0.7],
+        minLayers     = [3,5,5],
+        min3DLayers   = [1,2,2],
+        maxLostLayers = [4,3,2],
+        maxDz         = [0.5,0.35,0.2],
+        maxDr         = [0.3,0.2,0.1]
+	),
+    vertices = 'firstStepGoodPrimaryVertices'
+)
```

Comment:
```suggestion
jetCoreRegionalStepBarrel = jetCoreRegionalStep.clone(
    src = 'jetCoreRegionalStepBarrelTracks',
    mva = dict(
	# minPixelHits  = [1,1,1], they could be easily increased to at least 2 or 3 !
        min3DLayers   = [1,2,2],
	),
)
```
to update only where needed.
Since the comment, which is not present in `jetCoreRegionalStep` seems to be valid here, my suggestion is to still keep the line but just as a comment, since it's not really applying different selection